Clicker/Models/Course.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  Course.swift
+//  Clicker
+//
+//  Created by Kevin Chan on 9/26/17.
+//  Copyright © 2017 CornellAppDev. All rights reserved.
+//
+
+import UIKit
+
+enum SubjectType {
```

Comment:
Sadly Subject needs to be a String instead
I don't see a lot of benefit in making every possible subject an enum case.